static/js/pages/certificates/NoAuth.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from "react";
+import BaseLayout from "components/BaseLayout";
+import { Row, Col } from "@canonical/react-components";
+
+const NoAuth: FC = () => {
+  return (
+    <BaseLayout title="Client authentication needed">
+      <Row>
+        <Col size={12}>
+          <h5>Uh-oh! Looks like you are not authenticated.</h5>
+          <p>
+            <a href="/ui/certificates">
```

Comment:
The same question here about page reloading.

Reply:
See the other comment - the page is not the same.

Reply:
I deleted my other comment since I misunderstood your original question. It is safe to use `Link` here, I will update this part.